src/EditorFeatures/VisualBasicTest/MakeTypeAbstract/MakeTypeAbstractTests.vb

```diff
@@ -22,7 +22,7 @@ Public Class [|Foo|]
     Public MustOverride Sub M()
 End Class",
 "
-Public MustOverride Class Foo
```

Comment:
📝 The new analyzer test library would never have allowed this test to pass

/cc @jmarolf

Reply:
yeah, where are we on migrating everything to the new test framework? Do we have an issue filed somewhere?